docs/querying/examples.md

```diff
@@ -87,9 +89,11 @@ following for every instance:
 ...we could get the top 3 CPU users grouped by application (`app`) and process
 type (`proc`) like this:
 
-    topk(3, sum(rate(instance_cpu_time_ns[5m])) by (app, proc))
+    topk(3, sum by (app, proc) (rate(instance_cpu_time_ns[5m])))
 
 Assuming this metric contains one time series per running instance, you could
 count the number of running instances per application like this:
 
-    count(instance_cpu_time_ns) by (app)
+    count by (app) (
```

Comment:
No good reason to split this over multiple lines IMO...

Reply:
Sure, I guess I can put them back to single line. I prefer multi-line when doing more complex nested operations like `sum(rate())`.